src/main/java/org/embulk/parser/jsonpath/JsonpathParserPlugin.java

```diff
@@ -158,8 +160,14 @@ public void run(TaskSource taskSource, Schema schema,
             FileInputInputStream is = new FileInputInputStream(input);
             while (is.nextFile()) {
                 final JsonNode json;
-                try {
-                    json = JsonPath.using(JSON_PATH_CONFIG).parse(is).read(jsonRoot, JsonNode.class);
+                try (ByteArrayOutputStream bout = new ByteArrayOutputStream()) {
```

Comment:
BTW: 

How about the following solution (if you do not want to exec `close()` by `parse(is)`)?
Wrapping the original `is` into a temporary `InputStream` will avoid allocating(& copying) a new buffer to parse json.

```
final InputStream toParse = new InputStream() {
    @Override
    public int read() {
        return is.read();
    }
};

try  {
    json = JsonPath.using(JSON_PATH_CONFIG).parse(toParse).read(jsonRoot, JsonNode.class);
}
```

Reply:
@takumakanari 
Many thanks! I think that is the better solution.

Reply:
please take care of closing `is` after the operations 👍